src/components/dropdown-menu/DropdownMenu.js

```diff
@@ -47,25 +47,25 @@ class DropdownMenu extends React.Component<Props, State> {
     componentDidUpdate(prevProps: Props, prevState: State) {
         if (!prevState.isOpen && this.state.isOpen) {
             // When menu is being opened
-            document.addEventListener('click', this.handleDocumentClick, true);
-            document.addEventListener('contextmenu', this.handleDocumentClick, true);
+            document.addEventListener('click', this.handleDocumentClick, false);
```

Comment:
`false` is the default param for addEventListener.

Reply:
fixed